backend/test/integration/StateChangeEvents.integration.test.js

```diff
@@ -0,0 +1,254 @@
+const Client = require('socket.io-client');
+const SocketIOServer = require('../../index');
+const config = require('../../config.json');
+const CreateLobbyDTO = require('../../domain/DTO/request/CreateLobbyDTO');
+const JoinLobbyDTO = require('../../domain/DTO/request/JoinLobbyDTO');
+
+const port = process.env.PORT || config.local_port;
+
+let clientSockets = [];
+let lobbyCode = null;
+
+// Create a new client, and connect it to the server via a socket
+beforeEach(async (done) => {
+    // Setting lobby code field
+    lobbyCode = await connectAndCreateLobby();
```

Comment:
how does 'connectAndCreateLobby()` function work since you havent imported it?

Reply:
its on the bottom of the file, could be exported out if needed